src/expression-modifiers/normalization/properties.js

```diff
@@ -0,0 +1,138 @@
+              const { qDimensionInfo } = handler.layout.qHyperCube;
+              return qDimensionInfo.map((dim, idx) => ({
+                value: idx,
+                label: dim.qGroupFallbackTitles[0],
+              })); // To avoid depending on the layout, we use the first dimension in the drill down dimension
+            },
+            show(itemData, handler) {
+              return handler.layout.qHyperCube.qDimensionInfo.length > 1;
+            },
+          },
+          relativeNumbers: {
+            refFn: data => `${getRef(data, rootPath)}.relativeNumbers`,
+            type: 'string',
+            translation: translationKeys.modifierRelativeNumbers || 'properties.modifier.relativeNumbers',
+            title: {
+              translation:
+              translationKeys.modifierRelativeNumbersTooltip || 'properties.modifier.relativeNumbers.tooltip',
+            },
+            component: 'dropdown',
+            schemaIgnore: true,
+            defaultValue: NORMALIZATION_TYPES.RELATIVE_TO_TOTAL_SELECTION,
+            options: [
+              {
+                value: NORMALIZATION_TYPES.RELATIVE_TO_TOTAL_SELECTION,
+                translation: translationKeys.relativeNumbersTotalSelection || 'properties.modifier.relativeNumbers.total.selection',
+              },
+              {
+                value: NORMALIZATION_TYPES.RELATIVE_TO_DIM_UNIVERSE,
+                translation: translationKeys.relativeNumbersDimUniverse || 'properties.modifier.relativeNumbers.dimensional.universe',
+              },
+              {
+                value: NORMALIZATION_TYPES.RELATIVE_TO_TOTAL_UNIVERSE,
+                translation: translationKeys.relativeNumbersTotalUniverse || 'properties.modifier.relativeNumbers.total.universe',
+              },
+            ],
+          },
+          crossAllDimensions: {
+            refFn: data => `${getRef(data, rootPath)}.crossAllDimensions`,
+            type: 'boolean',
+            translation: translationKeys.crossAllDimensions || 'properties.modifier.crossAllDimensions',
+            title: {
+              translation:
+              translationKeys.crossAllDimensionsTooltip || 'properties.modifier.normalization.crossAllDimensions.tooltip',
+            },
+            schemaIgnore: true,
+            defaultValue: false,
+            show(itemData, handler) {
+              return handler.layout.qHyperCube.qDimensionInfo.length > 1;
+            },
+          },
+          showExcludedValues: {
+            refFn: data => `${getRef(data, rootPath)}.showExcludedValues`,
+            type: 'boolean',
+            translation: translationKeys.showExcludedValues || 'properties.modifier.showExcludedValues',
+            schemaIgnore: true,
+            defaultValue: true,
+          },
+        },
+        show(itemData, handler) {
+          return helper.isApplicable({ properties: handler.properties });
+        },
+      },
+    },
+    show(itemData) {
```

Comment:
Use the following to check if the properties should be shown or not
```
show(itemData, handler, args) {
    const modifierTypes = args.ext.support.modifiers;
    const supported = modifierTypes && Array.isArray(modifierTypes) && modifierTypes.indexOf(MODIFIER_TYPE);
	if(!supported){
		return false;
	}
    const modifier = getModifier(itemData, rootPath);
    return modifier && !modifier.disabled;
},
```

Reply:
Applied this code and made a change modifierTypes.indexOf(MODIFIER_TYPE) !== -1